raft/index.js

```diff
@@ -349,6 +349,13 @@ class Raft extends EventEmitter {
 
                 //
                 // RPC command
+                case 'rpc':
+                    //TODO Check this make sure that it is called through the leader of 
+                    console.log("RAHUL in command case :", raft.leader === this.address)
```

Comment:
Let's also convert all console logs to proper logging. [Pino](https://github.com/pinojs/pino) is a good alternative.